pkg/kubectl/cmd/drain.go

```diff
@@ -93,6 +94,8 @@ func NewCmdUncordon(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 			cmdutil.CheckErr(options.RunCordonOrUncordon(false))
 		},
 	}
+	cmdutil.AddInclude3rdPartyFlags(cmd)
```

Comment:
Removed.  In general, I initially added it to everything since it was needed for the Object(...) function, but I subsequently removed a bunch of them.  Feel free to indicate other places that need removal.